llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -94,9 +103,13 @@ def _create_prompt(task: Task, plan: Optional[Plan] = None) -> str:
             obj_type = domain.constants[obj]
             type_to_objs[obj_type].append(obj)
         objects_strs: List[str] = []
+        objs_dict = {}
         for typ, objs in type_to_objs.items():
             if not objs:
                 continue
+            if rng:
+                objs_dict = utils.randomize_object_names(rng, objs)
```

Comment:
it looks like this is going to get overwritten as the loop continues -- is that a bug?